notebooks/book1/02/softmax_plot.ipynb

```diff
@@ -1,55 +1,203 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    a = jnp.array(
</pre><p>I think you are using black with default settings, due to this this code (line 2 to 8) looks ugly.</p><p>You can use following commands to fix,</p><p><code>pip install black[jupyter]</code></p><p><code>black --verbose --line-length=120 path</code>/to/notebook&gt;</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/915/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Yes, understood the problem. I will update it.

Reply:
I didn't affect it.

![image](https://user-images.githubusercontent.com/52617262/174448704-871c3cd9-eb03-4e30-a913-5b576ce60ad6.png)

Reply:
I see it happened because this notebook is already passed by black on default settings, in this case, you could manually fix ugly code, i.e, keep all elements in the same line in Jax array